microsoft-edge/devtools-guide-chromium/resources/index.md

```diff
@@ -20,139 +20,175 @@ ms.date: 05/04/2021
    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
    See the License for the specific language governing permissions and
    limitations under the License.  -->
-# View page resources
+# View the resource files that make up a webpage
 
-Resources are the files that a page needs in order to display correctly.  Examples of resources include:
-*  CSS files.
-*  JavaScript files.
-*  HTML files.
-*  Image files.
+Resources are the files that make up a webpage.  Examples of resources include:
+*  CSS files
+*  JavaScript files
+*  HTML files
+*  Image files
 
-You can view the resources that make up a web page from within several tools, or panels.
+You can view the resources that make up a web page from within several tools (or _panels_) in Microsoft Edge DevTools, including:
+*  **Network** tool
+*  **Sources** tool
+*  **Application** tool
 
-This guide assumes that you are familiar with the basics of [web development](https://developer.mozilla.org/docs/Learn) and [Microsoft Edge DevTools](../../devtools-guide-chromium/overview.md).
+See also:
+*  [Learn web development](https://developer.mozilla.org/docs/Learn) at Mozilla.org
+*  [Overview of DevTools](../../devtools-guide-chromium/overview.md)
 
 
 <!-- ====================================================================== -->
-## Open resources from the Command Menu
+Open resource files from the Command Menu
 
-When you know the name of the resource that you want to inspect, the **Command Menu** provides a fast way of opening the resource.
+When you know the name of a webpage's resource file that you want to inspect, the **Command Menu** in DevTools provides a fast way of finding and opening that resource.
 
```

Comment:
```suggestion

1. In a new window or tab, go to a webpage, such as the [Accessibility-testing features demo](https://microsoftedge.github.io/Demos/devtools-a11y-testing/).  (The source files for that webpage are at [MicrosoftEdge / Demos > devtools-a11y-testing](https://github.com/MicrosoftEdge/Demos/tree/main/devtools-a11y-testing).)

```

Reply:
This Suggestion wasn't applied yet was Resolved.  Maybe this comment was accidentally resolved.  @dan-wesley